docker/docker-compose.license-proxy.yml

```diff
@@ -14,6 +14,7 @@ services:
     # Make sure you `export` your self-hosted API key secret in your local environment
     environment:
       DEEPGRAM_API_KEY: "${DEEPGRAM_API_KEY}"
+      DEEPGRAM_DEPLOYMENT_ORCHESTRATOR: "docker-compose"
```

Comment:
Any way to make this a static variable or enum of some type, or is hard-coding the same string multiple times the best option on these?

Reply:
I wasn't previously aware, which is why there is duplication in other places, but your question prompted me to go looking, and I found Docker Compose extension fields. Let me see if they're also compatible with Podman.

Reply:
The Compose protocol is fairly limited. I was able to reduce the duplication of the API key and the common orchestrator variable. I tried to do the same for the release tag on each image, but I can edit only a subsection of a field.

It's also not supported by `podman-compose` right now, so can only use this for the Docker Compose templates.